src/main.ts

```diff
@@ -1,4 +1,6 @@
-import { app, BrowserWindow, ipcMain } from 'electron';
+import { app, BrowserWindow } from 'electron';
+
+import { runService as runSigningService } from '@api/signing';
```

Comment:
Naming: use nouns. `@api/sign`, `SignService` etc. 
But since the app will eventually need to know which accounts are available in the Signer it would be the right time to add `WalletService`